### ISSUE TITLE
FR: add “Book status” or “Mark as…” to History and File Manager

### ISSUE THREAD

**Issue by mergen3107** [koreader/koreader#8954]
**Does your feature request involve difficulty completing a task? Please describe.**
Now that we have a very useful sorting mechanism in History by books on hold, I think it would be handy to be able to change the status from History and File Manager. 

**Describe the solution you'd like**
History’s context menu has Book description and Book information. Let’s add Book status as well. File Manager has “Mark as finished”, but not other statuses. 

**Describe alternatives you've considered**
The only way to modify these statuses is by going to Book status, but only after opening the book. 

**Additional context**
A use case maybe when the user opens and skims a few books and the decides to put some of them on hold, then sort History by “Reading” and focus on a few more important books first. 

This is not a deal breaking omission, but I think such improvement will make my beloved KOReader even more useful tool for reading books. 

Thank you!

**Comment by mergen3107** [comment]
Thank you everyone!!! 
Can’t wait to test!
